README.md

```diff
@@ -137,6 +137,18 @@ See [CHANGELOG.md](CHANGELOG.md)
 > However, the API is based on the Principle of Least Privilege, and exposes only a limited set of methods.
 > We aren't quite there yet, but we envision Talos being a great place for the application of [control theory](https://en.wikipedia.org/wiki/Control_theory) in order to provide a self-healing platform.
 
+**How is Talos different than CoreOS/RancherOS/Linuxkit?**
+
+> Talos is similar in many ways, but there are some differences that make it unique.
+> In some ways you can imagine Talos as a container image, in that it is built with a single purpose in mind.
+> In this case, that purpose is Kubernetes.
+> Talos tightly integrates with Kubernetes, and is not meant to be a general use Linux distribution.
+> This allows us to dramatically decrease the footprint of Talos, and in turn improve a number of other areas like security, and reliability.
+> In addition to this, interaction with the host is done through a gRPC API.
+> You might be wondering how you might run application X if you don't have access to the host.
+> Our answer would be Kubernetes!
+> Simply put, Talos makes a machine a Kubernetes node.
```

Comment:
Maybe "simply put" makes it seem like the previous sentence is what I'm summing up.